components/LinkCustom/LinkCustom.tsx

```diff
@@ -16,13 +16,13 @@ interface ILinkCustomProps {
  * @param props.href - this is the URL you want the link to point to
  * @returns The custom link component
  */
-const LinkCustom = ({ children, href }: ILinkCustomProps): JSX.Element => {
+const LinkCustom = ({ children, href, ... props }: ILinkCustomProps): JSX.Element => {
   return (
     <Link
-      className={'hover:text-orange-600 hover:underline'}
-      data-testid="linkCustom"
+      className={'font-bold text-orange-600 hover:text-orange-600 hover:underline'}
       href={href}
```

Comment:
Could you please explain why `data-testid="linkCustom` was deleted?

Reply:
It was hard coded that way, and for every link the value would be `linkCustom`. 
Initially I wanted to specifically destructure `data-testid` so TypeScript would let us know when it's missing. But looking at the `Button` component I noticed that it's bundled with the rest of the `... props` so I followed the same approach.